nxdrive/osi/windows/windows.py

```diff
@@ -41,6 +65,33 @@ def zoom_factor(self) -> float:
                 self.__zoom_factor = 1.0
         return self.__zoom_factor
 
+    @pyqtSlot(result=bool)
+    def addons_installed(self) -> bool:
+        """Check if add-ons are installed or not."""
+        return any(
```

Comment:
Hm to have lazy checks, I would rewrite to:
```python
if Options.system_wide or (Path(sys.executable).parent / "addons-installed.txt").is_file(): ...
```
Else, code in both conditions will be executed. Using the `if` will execute the 1st one only if it returns `True`.